project/src/main/webapp/index.html

```diff
@@ -13,81 +13,81 @@
-        <label for="cuisine-homemade">Homemade</label>
-      </form>
-      <form id="rating-form" class="form-element-toolbar">
-        <p>Desired rating (out of 5):</p>
-        <input type="radio" id="rating-0" name="rating" value="1" checked="checked">
-        <label for="rating-0">I don't mind</label>
-        <input type="radio" id="rating-1" name="rating" value="1">
-        <label for="rating-1">1 star</label>
-        <input type="radio" id="rating-2" name="rating" value="2">
-        <label for="rating-2">2 star</label>
-        <input type="radio" id="rating-3" name="rating" value="3">
-        <label for="rating-3">3 star</label>
-        <input type="radio" id="rating-4" name="rating" value="4">
-        <label for="rating-4">4 star</label>
-        <input type="radio" id="rating-5" name="rating" value="5">
-        <label for="rating-5">5 star</label>
-      </form>
-      <form id="price-form" class="form-element-toolbar">
-        <p>Desired price level:</p>
-        <input type="radio" id="price-all" name="price" value="4" checked="checked">
-        <label for="price-all">I don't mind</label>
-        <input type="radio" id="price-1" name="price" value="1">
-        <label for="price-1">Inexpensive</label>
-        <input type="radio" id="price-2" name="price" value="2">
-        <label for="price-2">Moderate</label>
-        <input type="radio" id="price-3" name="price" value="3">
-        <label for="price-3">Expensive</label>
-        <input type="radio" id="price-4" name="price" value="4">
-        <label for="price-4">Very Expensive</label>
-      </form>
-      <form id="open-now-form" class="form-element-toolbar">
-        <p>Should it be open now?</p>
-        <input type="radio" id="open-now-yes" name="open-now" value="1" checked="checked">
-        <label for="open-now-yes">Yes</label>
-        <input type="radio" id="open-now-dont-mind" name="open-now" value="0">
-        <label for="open-now-dont-mind">I don't mind</label>
-      </form>
-      <div id="input-error-container" class="error-text"></div>
-    </div>
-    <div id="user-map-container">
-      <br> <br>
-      <form id="location-form" class="form-element-toolbar">
-        <input type="radio" id="device-location" class="device-location" name="location"
-        value="device-location" onclick="getDeviceLocationAndShowOnMap()">
-        <label for="device-location">Press here to use your device's location!</label>
-        <div id="map-warning-container">Please note: Mac devices might display an inaccurate location.
-          <br> It is recommended for Mac users to use the search box.</div>
-      </form>
-      <input id="location-input" class="controls" type="text" placeholder="Search Box"/>
-      <div id="map-error-container" class="error-text"></div>
-      <div id="map"></div>
-      <script> addMapWithSearchBox() </script>
-     </div>
+      <div id="sign-in-segment">
```

Comment:
I don't know why this happened, but almost all lines are marked as changed while I think only a few lines were added. It is pretty hard to review this way so maybe if you can revert to the last version and just add the lines and tabs that you wanted to add in this PR it can be great.

If it's too much time to do that now, you can leave it like this, but can you point out the changes? Only "user-input" class was added right?

Reply:
It happened because the big divs changed. sadly even if we revert it, it will still look the same here, because git doesn't handle html very well.